selectors/emojis.js

```diff
@@ -39,18 +41,15 @@ export const getRecentEmojis = createSelector(
     },
 );
 
-function getStateRecentSkin(state) {
-    return state.views.emoji?.recentSkin;
-}
-
-export function getRecentSkin(state) {
-    const stateSkin = getStateRecentSkin(state);
-    if (stateSkin) {
-        return stateSkin;
-    }
-    const recentSkin = LocalStorageStore.getRecentSkin(getCurrentUserId(state));
-    return recentSkin || 'default';
-}
+// todo rename to getEmojiSkin
```

Comment:
Did you want to address this before this PR goes in?

Reply:
i did rename all "recentSkin" but went with a different name at the end, I just forgot about this comment which was left in case I forgot to do it.